spec/tictactoe_spec.rb

```diff
@@ -0,0 +1,142 @@
+require 'board'
+require 'game'
+
+RSpec.describe Game do
+  subject(:game) { Game.new(Array.new(9, ' ')) }
+  describe '#initialize' do
+    it 'should be initialized with an array of 9 empty strings' do
+      expect(game.table).to eq(Array.new(9, ' '))
+    end
+  end
+
+  describe '::valid_letter?' do
+    subject { Game.valid_letter?(letter) }
+    let(:letter) { 'D' }
+    it 'should accept a char (string) as an argument' do
+      expect(letter).to be_kind_of String
```

Comment:
You should also check the negative case (not be) if the user inputs something else than a string.